### DISCUSSION TITLE
refactor getter method to get property

### DISCUSSION THREAD

**Discussion by mick62** [nicoespeon/abracadabra#339] [Ideas]
Hi,

another useful (and needed) refactoring is the change from:

class A {
  getProp(): Y
     { return expr }
}
x = myA.getProp()

to

class A {
  get prop(): Y
     { return expr }
}
x = myA.prop

Btw. the counterpart 'setProp(val)' to 'set prop(val)' 
might also be of interest although we use immutable objects most of the time.

Thank you in advance

**Comment by nicoespeon** [comment]
I think that's a very good suggestion, thanks @mick62 

The tricky bit (especially in JS) will be to find and update all references to the method. Something that may require #275 to be addressed first maybe. 

In any case, I'm turning this into an issue to track the interest 👍
